mav_msgs/include/mav_msgs/default_topics.h

```diff
@@ -52,6 +52,7 @@ static constexpr char POSITION[] = "position";
 // Simulation-specific topic names.
 static constexpr char WRENCH[] = "wrench";
 static constexpr char WIND[] = "wind";
+static constexpr char WIND_SPEED[] = "wind_speed";
```

Comment:
What is the difference between wind and wind_speed? Maybe we can deprecate wind?